lesson_3/task_16.py

```diff
@@ -0,0 +1,21 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое
+# число X в массиве A[1..N]. Пользователь в первой строке вводит
+# натуральное число N – количество элементов в массиве. В последующих
+# строках записаны N целых чисел Ai. Последняя строка содержит число X
+#
+# *Пример:*
+#
+# 5
+#     1 2 3 4 5
+#     3
+#     -> 1
+
+import random
+n = int(input())
+x = int(input())
+a = [random.randint(1, n) for _ in range(n)]
+print(*a)
+print(a)
+print(a.count(int(x)))
```

Comment:
Алина, у вас переменная `х` уже в целочисленном типе и переводить её не нужно.
```suggestion
print(a.count(x))
```